packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -80,7 +90,11 @@ class NetworkRequestInspector extends StatelessWidget {
             if (data is DartIOHttpRequestData) ...[
               HttpRequestHeadersView(data),
               if (data.requestBody != null) HttpRequestView(data),
-              if (data.responseBody != null) HttpResponseView(data),
+              if (data.responseBody != null)
+                HttpResponseView(
+                  data,
+                  currentResponesNotifier: controller.currentResponseViewType,
```

Comment:
nit: spelling - currentResponseNotifier, and maybe consider renaming to currentResponseViewType for consistency.